addon/helpers/truncate-text.js

```diff
@@ -0,0 +1,9 @@
+import { helper } from '@ember/component/helper';
+
+export function truncateText([input], hash = {}) {
```

Comment:
Instead of creating a helper you can use `truncate` from [ember-string-helpers](https://github.com/romulomachado/ember-cli-string-helpers#truncate)
Like: https://github.com/ilios/common/blob/7553a10c3f8b7cd6a291a8ae316f9ff244185065/addon/templates/components/sessions-grid-offering.hbs#L60-L63

Reply:
👍